lib/views/pages/post/post_page.dart

```diff
@@ -83,10 +83,11 @@ class PostPage extends HookConsumerWidget {
           padding:
               // This is necessary to prevent the keyboard from covering the bottom bar
               EdgeInsets.only(bottom: MediaQuery.of(context).viewInsets.bottom),
-          child: const BottomBarAddComment(
+          child: BottomBarAddComment(
             key: bottomBarAddCommentKey,
             //TODO: Replace with actual username
             currentDisplayName: "Username",
```

Comment:
I agree with you.
However I believe that @CHOOSEIT is already working on it in #227 thus I prefer to avoid making modifications in this PR to reduce merge conflicts.